pkg/apis/auditregistration/v1alpha1/defaults.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	auditregistrationv1alpha1 "k8s.io/api/auditregistration/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	// DefaultQPS is the default QPS value
+	DefaultQPS = int64(10)
+	// DefaultBurst is the default burst value
+	DefaultBurst = int64(15)
```

Comment:
I suggest making these constant to 100% prevent the problem in the following comment.

Reply:
unfortunately we can't take the address of a constant so I'll have to go with the solution above

Reply:
Yes, but taking their address is a bug. Making them constants prevents the bug :)

Reply:
Oh, hold on a second, I'm too myopic-- I suggest actually either deleting this DefaultThrottle variable or making it a function which returns a shiny new object each time.

Reply:
o I was just referring to the API requiring those values to be pointers, so we would need a function anyway. Updating DefaultThrottle as well